modules/cardano-explorer.nix

```diff
@@ -1,3 +1,10 @@
+let
+
+  explorer' = (import ./../default.nix {}).cardano-sl-explorer-static;
+  explorer  = builtins.trace "using cardano-sl-explorer ${explorer'}" explorer';
```

Comment:
The answer is that this actually came up when a developer doubted that things were deployed as we claim them to be.